Resources/Prototypes/Entities/Objects/Devices/translator_implants.yml

```
@@ -0,0 +1,132 @@
+- type: entity
+  abstract: true
+  id: BaseTranslatorImplanter
+  parent: [ BaseItem ]
+  name: Basic translator implant
+  description: "Translates speech."
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/implanter.rsi
+    state: implanter0
+    layers:
+    - state: implanter1
+      map: [ "implantFull" ]
+      visible: true
+    - state: implanter0
+      map: [ "implantBroken" ]
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.ImplanterVisuals.Full:
+        implantFull:
+          True: {visible: true}
+          False: {visible: false}
+        implantBroken:
+          True: {visible: false}
+          False: {visible: true}
+
+- type: entity
+  id: BasicGalaticCommonTranslatorImplanter
+  parent: [ BaseTranslatorImplanter ]
+  name: Basic Galatic Common translator implant
+  description: "An implant giving the ability to understand Galatic Common."
+  components:
+  - type: TranslatorImplanter
+    understood:
+    - GalacticCommon
+
+- type: entity
+  id: AdvancedGalaticCommonTranslatorImplanter
+  parent: [ BaseTranslatorImplanter ]
+  name: Advanced Galatic Common translator implant
```

Spotted this while testing
```suggestion
  name: Advanced Galactic Common translator implant
```

---

Classic me and my grammar.